vendor.conf

```diff
@@ -38,7 +38,7 @@ github.com/ostreedev/ostree-go aeb02c6b6aa2889db3ef62f7855650755befd460
 github.com/pborman/uuid master
 github.com/pkg/errors master
 github.com/pquerna/ffjson d49c2bc1aa135aad0c6f4fc2056623ec78f5d5ac
-github.com/projectatomic/libpod e686269da34ed4208f4ed517c0587ab38e8eaf2c
+github.com/projectatomic/libpod 781eec27b52c842fc83c8b1c97fbf825065f3b0c
```

Comment:
Gonna show my golang vendoer ignorance here.  Can we instead vendor in github.com/projectatomic/libpod/pkg?  That seems to be the only code we use from libpod and I just wonder if we could set up some kind of circular nastiness if we vendor in all of libpod as it vendors in Buildah.

Reply:
vndr is smart enough to only pull in the parts that it needs. So unless something in libpod imports certain code from buildah, it will not end up in the vendor.